webapp/src/Form/Type/ContestType.php

```diff
@@ -25,12 +25,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $this->addExternalIdField($builder, Contest::class);
         $builder->add('shortname', TextType::class);
-        $builder->add('name', TextType::class);
+        $builder->add('name', TextType::class, [
+            'help' => 'Contest name as shown in the top right.'
```

Comment:
Also added this one here.

Reply:
I think this helptext applies to 'shortname', and 'name' is 'Contest name in full as shown on the scoreboard'